<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewTeacherDialog</class>
 <widget class="QDialog" name="ViewTeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>863</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Teacher</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>561</width>
     <height>781</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Personal Data</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Address</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Expenses</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Mobile</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="labName">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>271</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labAddress">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>271</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labPhone">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labMobile">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>140</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labDepart">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>180</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>230</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string># Hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="labHours">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>220</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labExpense">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>0</red>
           <green>0</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>144</red>
           <green>141</green>
           <blue>139</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>147</green>
           <blue>145</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="text">
      <string>Profit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Debt</string>
     </property>
    </widget>
    <widget class="QLabel" name="labDebt">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>330</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="palette">
      <palette>
       <active>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>4</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>4</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="WindowText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>144</red>
           <green>141</green>
           <blue>139</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Text">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>169</red>
           <green>167</green>
           <blue>167</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="ButtonText">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>150</red>
           <green>147</green>
           <blue>145</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>12</string>
     </property>
    </widget>
    <widget class="QLabel" name="labProfile">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>410</y>
       <width>251</width>
       <height>251</height>
      </rect>
     </property>
     <property name="text">
      <string>Profil</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>280</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Reg Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="labMemberSince">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="labEndMember">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>270</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>280</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>End Date</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabFunds">
    <attribute name="title">
     <string>Funds History</string>
    </attribute>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineStart">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEnd">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushFilter">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius:8px;
border-width:1px;
border-color: blue;
border-style: solid;
border-width:1px;
color :white;
background-color:  rgb(90,90, 90);
</string>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddAmount">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>700</y>
       <width>84</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  border-radius : 10px;
  color:rgb(255,128,0);
  border: 2px solid rgb(30,195,255);
  min-width: 80px;
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))
}


QPushButton:hover {
  border-color: navy; /* make the default button prominent */

}


QPushButton:pressed {
  border-radius : 10px;

  color:rgb(255,255,255);

  border: 2px solid rgb(30,195,255);
  min-width: 80px;

  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))

}


</string>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_24">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>700</y>
       <width>64</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linePoso">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>700</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>441</width>
       <height>571</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Table</string>
      </attribute>
      <widget class="QTableView" name="tableFunds">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>411</width>
         <height>481</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="comboWhatToShow">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>291</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>69</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Show</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Graph</string>
      </attribute>
     </widget>
    </widget>
    <widget class="QLabel" name="label_68">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>630</y>
       <width>71</width>
       <height>61</height>
      </rect>
     </property>
     <property name="text">
      <string>Salary</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="linenewSal">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>650</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushUpdateSal">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>650</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  border-radius : 10px;
  color:rgb(255,128,0);
  border: 2px solid rgb(30,195,255);
  min-width: 80px;
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))
}


QPushButton:hover {
  border-color: navy; /* make the default button prominent */

}


QPushButton:pressed {
  border-radius : 10px;

  color:rgb(255,255,255);

  border: 2px solid rgb(30,195,255);
  min-width: 80px;

  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))

}


</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineComments">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>700</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabHours">
    <attribute name="title">
     <string>Hours</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>End</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushFilterHours">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  border-radius : 10px;
  color:rgb(255,128,0);
  border: 2px solid rgb(30,195,255);
  min-width: 80px;
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))
}


QPushButton:hover {
  border-color: navy; /* make the default button prominent */

}


QPushButton:pressed {
  border-radius : 10px;

  color:rgb(255,255,255);

  border: 2px solid rgb(30,195,255);
  min-width: 80px;

  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))

}


</string>
     </property>
     <property name="text">
      <string>Filter</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_23">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>100</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
    </widget>
    <widget class="QTableView" name="tableHours">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>341</y>
       <width>511</width>
       <height>391</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_25">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>61</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Groups</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboTeachGroups">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>311</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkShowActive">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>341</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Show only Active Groups</string>
     </property>
     <property name="tristate">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QListWidget" name="listGroupMembers">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>190</y>
       <width>501</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelCourse">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>110</y>
       <width>301</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Available</string>
    </attribute>
    <widget class="QTableView" name="tableAvailable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>671</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::NoSelection</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushUpdateAvailable">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>700</y>
       <width>84</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {
  border-radius : 10px;
  color:rgb(255,128,0);
  border: 2px solid rgb(30,195,255);
  min-width: 80px;
  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))
}


QPushButton:hover {
  border-color: navy; /* make the default button prominent */

}


QPushButton:pressed {
  border-radius : 10px;

  color:rgb(255,255,255);

  border: 2px solid rgb(30,195,255);
  min-width: 80px;

  background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(45, 45,45, 255), stop:1 rgba(85, 85, 85, 255))

}


</string>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>Teaching</string>
    </attribute>
    <widget class="QTableView" name="tableTeaching">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>521</width>
       <height>621</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboAllCourses">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>291</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboDeps">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>231</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_69">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Department</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_83">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Course</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushAddCourse">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>84</width>
       <height>25</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>DaysOff</string>
    </attribute>
    <widget class="QTableView" name="tableOff">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>511</width>
       <height>701</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>421</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
